src/v/cluster/tx_gateway_frontend.cc

```diff
@@ -1480,17 +1480,17 @@ tx_gateway_frontend::do_commit_tm_tx(
             }
         }
 
-        std::vector<ss::future<prepare_group_tx_reply>> pgfs;
```

Comment:
> cluster: make txn coordinator ignore group prepare

can you add a sentence or two to the commit message that explains why it should be ignored?